.travis.yml

```diff
@@ -54,164 +66,197 @@ jobs:
     - if: type = cron
+      os: linux
       dist: trusty
       compiler: gcc
       stage: test
       name: "14.04 gcc vltmt1"
       script: ci/test.sh vltmt1
 #
     - if: type = cron
+      os: linux
       dist: xenial
       compiler: gcc
       stage: vbuild
       name: "16.04 gcc build"
       script: echo
     - if: type = cron
+      os: linux
       dist: xenial
       compiler: gcc
       stage: test
       name: "16.04 gcc distvlt"
       script: ci/test.sh distvlt
     - if: type = cron
+      os: linux
       dist: xenial
       compiler: gcc
       stage: test
       name: "16.04 gcc vltmt0"
       script: ci/test.sh vltmt0
     - if: type = cron
+      os: linux
       dist: xenial
       compiler: gcc
       stage: test
       name: "16.04 gcc vltmt1"
       script: ci/test.sh vltmt1
 #
     - if: type = cron
+      os: linux
       dist: bionic
       compiler: gcc
       stage: vbuild
       name: "18.04 gcc build"
       script: echo
     - if: type = cron
+      os: linux
       dist: bionic
       compiler: gcc
       stage: test
       name: "18.04 gcc distvlt"
       script: ci/test.sh distvlt
     - if: type = cron
+      os: linux
       dist: bionic
       compiler: gcc
       stage: test
       name: "18.04 gcc vltmt0"
       script: ci/test.sh vltmt0
     - if: type = cron
+      os: linux
       dist: bionic
       compiler: gcc
       stage: test
       name: "18.04 gcc vltmt1"
       script: ci/test.sh vltmt1
 #
     - if: type = cron
+      os: linux
       dist: xenial
       compiler: clang
       stage: vbuild
       name: "16.04 clang build"
       script: echo
     - if: type = cron
+      os: linux
       dist: xenial
       compiler: clang
       stage: test
       name: "16.04 clang distvlt"
       script: ci/test.sh distvlt
     - if: type = cron
+      os: linux
       dist: xenial
       compiler: clang
       stage: test
       name: "16.04 clang vltmt0"
       script: ci/test.sh vltmt0
     - if: type = cron
+      os: linux
       dist: xenial
       compiler: clang
       stage: test
       name: "16.04 clang vltmt1"
       script: ci/test.sh vltmt1
 #
     - if: type = cron
+      os: linux
       dist: focal
       compiler: clang
       stage: vbuild
       name: "20.04 clang build"
       script: echo
     - if: type = cron
+      os: linux
       dist: focal
       compiler: clang
       stage: test
       name: "20.04 clang distvlt"
       script: ci/test.sh distvlt
     - if: type = cron
+      os: linux
       dist: focal
       compiler: clang
       stage: test
       name: "20.04 clang vltmt0"
       script: ci/test.sh vltmt0
     - if: type = cron
+      os: linux
       dist: focal
       compiler: clang
       stage: test
       name: "20.04 clang vltmt1"
       script: ci/test.sh vltmt1
+#
+    - if: type != cron
+      os: freebsd
+      compiler: clang
+      stage: vbuild
+      name: "freebsd clang build and examples"
+      script: gmake examples
```

Comment:
Please use the separate 'test' stage to actually invoke the tests similar to the ones above. The 'vbuild' stage is there to reset the 50 minute timer after building verialtor itselt  in before_script. This should fix the current failure.